.github/workflows/clear-pr-caches.yml

```diff
@@ -1,17 +1,24 @@
 name: clear-pr-caches
 
 on:
-  pull_request:
+  pull_request_target:
     types: [ closed ]
 
 permissions:
```

Comment:
Should we move the permissions back to the job level with this, or is it required to be at the job level?

Reply:
Not required. I don't have a preference here, they are probably a bit more visible this way.